README.md

```diff
@@ -12,10 +12,10 @@ Automatically generated documentation of `django-security` is available on Read
 
 # Requirements
 
-* Python >= 2.7
-* Django >= 1.8
+* Python >= 3.6
+* Django >= 1.11
 
-For Django < 1.8 use django-security==0.9.4.
+For Django < 1.11 use django-security==0.9.5. For Django < 1.8 use django-security==0.9.4.
```

Comment:
Out of curiosity, why is it 0.9.5 for Django < 1.11? Does our current version support Django 1.8+?

Also strange thing I don't see 0.9.4 on PyPI.

Reply:
It actually should be `0.11.3` for Django < 1.11, as that's the last version that will support some of the older version. Fixed to it.

That's weird about 0.9.4. Let me take a look into it.